system/networking/connman-manual-proxy.py

```diff
@@ -0,0 +1,35 @@
+"""Ask connman for the manual proxy of the service that connects automatically.
+"""
+
+import dbus
+import time
+
+bus = dbus.SystemBus()
+
+client = dbus.Interface(
+    bus.get_object("net.connman", "/"),
+    "net.connman.Manager")
+
+def wait_for_connman():
+    # Waiting for connman, as it does not return the services until some
+    # seconds after startup (~4 seconds).
+    for i in range(0, 20):
+        if client.GetServices():
+            break
+        else:
+            time.sleep(0.5)
+
+wait_for_connman()
```

Comment:
If I remember correctly, this was not necessary when using `pacrunner`. I am not very happy with this solution.

Reply:
This code has been removed, in favor of monitoring proxy changes.